package.json

```diff
@@ -14,37 +14,41 @@
     ".prettierrc.js"
   ],
   "scripts": {
+    "prepare": "husky install",
+    "test": "npm run lint",
```

Comment:
[note] This `test` script is useful for the [`npm it`](https://docs.npmjs.com/cli/v9/commands/npm-install-test) command.